applications/zigbee_weather_station/README.rst

```diff
@@ -0,0 +1,512 @@
+
+#. When the device joins the network, **LED (LD1)** lights up with constant blue color.
+   Additionally, with the ``debug`` build type and console used for logging, output similar to the following one appears:
+
+   .. code-block:: console
+
+      [00:00:02.029,174] <inf> zigbee_app_utils: Zigbee stack initialized
+      [00:00:02.035,003] <inf> zigbee_app_utils: Device started for the first time
+      [00:00:02.035,003] <inf> zigbee_app_utils: Start network steering
+      [00:00:02.035,064] <inf> zigbee_app_utils: Started network rejoin procedure.
+      [00:00:05.637,512] <inf> zigbee_app_utils: Joined network successfully (Extended PAN ID: f4ce3655fc08bf6b, PAN ID: 0xbf9e)
+
+   .. note::
+      As timestamps in this snippet indicate, these logs are likely to appear before the periodic sensor or attribute logs.
+
+#. Read the weather attributes from the development kit programmed with the Zigbee shell sample (the network coordinator):
+
+   a. Run the following command to read the temperature attribute:
+
+      .. code-block:: console
+
+         uart:~$ zcl attr read 0x1ca9 42 0x0402 0x0104 0x0000
+
+      In this command:
+
+      * ``0x1ca9`` is the short network address of Thingy:53.
+      * ``42`` is the endpoint number for aggregating the clusters.
+      * ``0x0402`` is the temperature cluster ID.
+      * ``0x0104`` is the Zigbee Home Automation profile ID.
+      * ``0x0000`` is the short network address of the coordinator device.
+
+      The output will be similar to the following one:
+
+      .. code-block:: console
+
+         ID: 0 Type: 29 Value: 2394
+
+      Here, the value ``2394`` means 23.94 degrees Celsius.
+      In Wireshark, the corresponding response frame for this output looks like follows:
+
+      .. code-block::
+
+         ZigBee HA	2298	4756.825525	0x1ca9	0x0000	0x1ca9	0x0000	f4:ce:36:23:d7:4e:77:1d	f4:ce:36:55:fc:08:bf:6b		0xbf9e	ZCL: Read Attributes Response, Seq: 0
+
+         Frame 2298: 68 bytes on wire (544 bits), 68 bytes captured (544 bits)
+         IEEE 802.15.4 Data, Dst: 0x0000, Src: 0x1ca9
+         ZigBee Network Layer Data, Dst: 0x0000, Src: 0x1ca9
+         ZigBee Application Support Layer Data, Dst Endpt: 64, Src Endpt: 42
+         ZigBee Cluster Library Frame, Command: Read Attributes Response, Seq: 0
+            Frame Control Field: Profile-wide (0x18)
+            Sequence Number: 0
+            Command: Read Attributes Response (0x01)
+            Status Record
+               Attribute: Measured Value (0x0000)
+               Status: Success (0x00)
+               Data Type: 16-Bit Signed Integer (0x29)
+               Measured Value: 23.94 [°C]
+
+   #. Run the following command to read the pressure attribute:
+
+      .. code-block:: console
+
+         uart:~$ zcl attr read 0x1ca9 42 0x0403 0x0104 0x0000
+
+      Compared with the previous command, ``0x0403`` in this command stands for the pressure cluster ID.
```

Comment:
```suggestion
      Compared with the previous command, ``0x0403`` in this command represents for the pressure cluster ID.
```

Reply:
If you agree with this suggestion, then please apply to lines 354, 407, and 414 as well.